nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -331,10 +342,27 @@ buildBuffers(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
 flatbuffers::Offset<
   flatbuffers::Vector<flatbuffers::Offset<tflite::OperatorCode>>>
 buildOperatorCodes(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
-  /** NYI! */
-  return flatbuffers::Offset<
-    flatbuffers::Vector<flatbuffers::Offset<tflite::OperatorCode>>>();
-};
+  const auto &op_codes = map.getIndexMap<tflite::BuiltinOperator>().getData();
+
+  std::vector<flatbuffers::Offset<tflite::OperatorCode>> fb_op_codes;
+  fb_op_codes.reserve(op_codes.size());
+
+  auto create_op_offset = [&fbb](const tflite::BuiltinOperator &op,
+                                 int32_t version = 1) {
+    tflite::OperatorCodeBuilder builder(fbb);
+    builder.add_deprecated_builtin_code(static_cast<int8_t>(op));
+    /// @todo find reason why version field is not shown
```

Comment:
what are the possible values of the version? is it possible to get this version from tflite::interpreter?

Reply:
It's been a while after I checked the source code but I remember it was hardcoded to 1 in tf sourcefile as well.

Reply:
then lets keep track to their code if they update it, and let it be 1 for now :)